packages/account/src/Sections/Profile/PersonalDetails/personal-details.jsx

```diff
@@ -155,7 +155,17 @@ export const PersonalDetailsForm = ({
             getSettings();
         }
         initializeFormValues();
-    }, [account_settings, is_eu, is_mf, is_social_signup]);
+    }, [
+        account_settings,
+        is_eu,
+        is_mf,
+        is_social_signup,
+        fetchResidenceList,
+        fetchStatesList,
+        has_residence,
+        isMounted,
+        setIsStateLoading,
+    ]);
```

Comment:
For `useEffect` deps please add to sanity check in QA checklist. This might cause some issues like multiple re-renders etc

Reply:
thank u for your revision sure I'll add that